arm/Microsoft.Network/virtualNetworkGateways/deploy.bicep

```diff
@@ -290,7 +288,7 @@ resource virtualGatewayPublicIP 'Microsoft.Network/publicIPAddresses@2021-02-01'
     publicIPPrefix: ((!empty(publicIPPrefixId)) ? publicIPPrefix : json('null'))
     dnsSettings: ((length(virtualGatewayPipName_var) == length(domainNameLabel)) ? json('{"domainNameLabel": "${domainNameLabel[index]}"}') : json('null'))
   }
-  zones: publicIpZones
+  zones: contains(zoneRedundantSkus, virtualNetworkGatewaySku) ? publicIpZones : json('null')
```

Comment:
Hey @torivara just one comment, please change `json('null')` to just `null`.

Reply:
Bicep magic ;)

Reply:
I was just following the rest of the document, @MrMCake. Should all json('null') be changed to just null? There are 19 occurences of json('null') the vnetgateway bicep file alone.

Reply:
@torivara  If so, please do, yes. The more consistent the code is the better. Will check the repo in general today to make sure this happens in all other places too.
We're saving ink ;)

Reply:
606 results in 87 files. Nice. Will do that after we merged your PR

Reply:
Search/Replaced all json('null') with null now. This looks like it is working. Tested deployment to my own sub, and there are no validation errors. Thank you for the suggestion!